scripts/exrrfs_ioda_bufr.sh

```diff
@@ -35,9 +31,19 @@ for yaml in ${yaml_list[@]}; do
  ${MPI_RUN_CMD} ${execfile} ${yaml}
  # some data may not be available at all cycles, so we don't check whether bufr2ioda.x runs successfully
 done
-ls ./ioda*nc  
+
+# run offline IODA tools
+${cpreq} ${HOMErrfs}/sorc/RDASApp/rrfs-test/IODA/offline_add_var_to_ioda.py .
+ioda_files=$(ls ioda*nc)
+for ioda_file in ${ioda_files[@]}; do
+  offline_add_var_to_ioda.py -o ${ioda_file}
```

Comment:
Sorry for another comment, could you add `./` before `offline_add_var_to_ioda.py`? 
Not all environments put the current directory in the $PATH variable.
_(for example, our Jet role account cron jobs set $PATH to empty)_